### PR TITLE
Remove support module.

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Review by mbohlkeschneider — COMMENTED** [review]
Thank you!

Why do we want to get rid of the support module? Do I understand the change right that the support functions have been factored out and put into their respective modules where they fit (for example `pad_to_size` to `mx/batchify`)?

**Comment by jaheba** [comment]
> Why do we want to get rid of the support module? Do I understand the change right that the support functions have been factored out and put into their respective modules where they fit (for example `pad_to_size` to `mx/batchify`)?

The support module is/was a collection of more or less unrelated things. So, yes, my goal is to move the functions to where they are actually used.

**Review by mbohlkeschneider — APPROVED** [review]

File: src/gluonts/transform/convert.py [hunk 1]
```diff
@@ -27,6 +26,82 @@
 )
 
 
+def erf(x: np.array) -> np.array:
```
Comment (mbohlkeschneider) [inline src/gluonts/transform/convert.py:29]: This one seems to have only one specific usage throughout the code and I think one could argue that it should be a function for the class that uses it, no? Not sure about other cases here.

Comment (jaheba) [reply src/gluonts/transform/convert.py:29]: I think it's fine to have them as standalone functions, since `erf` isn't tied to some object state.

**Review by mbohlkeschneider — APPROVED** [review]

**Pull request merged**